docs/source/java/install.rst

```diff
@@ -28,15 +28,15 @@ Java modules are regularly built and tested on macOS and Linux distributions.
 Java Compatibility
 ------------------
 
-Java modules are currently compatible with Java 8 / 9 / 10 / 11.
+Java modules are currently compatible with JDK 8, 9, 10, or 11, but only JDK 11 is tested in CI.
 
 Installing from Maven
 ---------------------
 
 By default, Maven will download from the central repository: https://repo.maven.apache.org/maven2/org/apache/arrow/
 
 Configure your pom.xml with the Java modules needed, for example:
-``arrow-memory-netty``, ``arrow-format``, and ``arrow-vector``.
+arrow-vector, arrow-memory-netty and flight-core.
```

Comment:
```suggestion
arrow-vector, arrow-memory-netty, and flight-core.
```

Reply:
That said: I would rather we show a separate POM with the changes needed for Flight, including os-maven-plugin.

Reply:
Added